demo

```diff
@@ -81,9 +81,17 @@ demo.run() {
   demo.logs
 }
 
+demo.help() {
+  echo "USAGE:"
+  echo "demo prep           creates project and service account"
+  echo "demo setup          creates tasks, pipeline and resources"
+  echo "demo run            starts pipeline"
+}
+
 
 main() {
   local fn="demo.${1:-''}"
+  test "$fn" = "demo.''" && demo.help && exit 0
```

Comment:
why do we need this? wouldn't it work without this line?

Reply:
Updated to remove that line